torch/distributed/device_mesh.py

```diff
@@ -367,14 +382,14 @@ def __eq__(self, other: object) -> bool:
                     and self._thread_id == other._thread_id
                 )
 
-        def __getitem__(self, mesh_dim_name: str) -> "DeviceMesh":
+        def __getitem__(self, mesh_dim_names: Union[str, Tuple[str]]) -> "DeviceMesh":
```

Comment:
this typing is incorrect.  It should be `Tuple[str, ...]` to allow for variable length tuples of strings.  Currently, it will only pass linter if called with single dim tuple.